apps/src/puzzleRatingUtils.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @overview A helper class for all actions associated with Puzzle
+ * Ratings, aka the Fun-O-Meter.
+ */
+
+var PuzzleRatingUtils = {};
+
+module.exports = PuzzleRatingUtils;
+
+var dom = require('./dom');
+
+/**
+ * Construct the puzzle rating buttons themselves
+ *
+ * @returns {Element} div containing puzzle ratng buttons with attached
+ *          click handlers
+ */
+PuzzleRatingUtils.buildPuzzleRatingButtons = function () {
+  var buttonContainer = document.createElement('div');
+  buttonContainer.id = 'puzzleRatingButtons';
+  buttonContainer.innerHTML = require('./templates/puzzleRating.html.ejs')();
+
+  var buttons = buttonContainer.querySelectorAll('.puzzle-rating-btn');
+  var buttonClickHandler = function () {
+    for (var i = 0, button; (button = buttons[i]); i++) {
+      if (button != this) {
+        $(button).removeClass('enabled');
+      }
+    }
+    $(this).toggleClass('enabled');
+  };
+  for (var i = 0, button; (button = buttons[i]); i++) {
+    dom.addClickTouchEvent(button, buttonClickHandler);
+  }
+
+  return buttonContainer;
+};
```

Comment:
This seems like it could be a good thing to do in React. Any reason we chose not to? Not sure whether it's worth converting at this point (but if you're looking for an opportunity to get to do some React, I would encourage it :))